server/server.go

```diff
@@ -25,6 +25,8 @@ import (
 	"github.com/elastic/go-elasticsearch/v7"
 	"github.com/elastic/go-elasticsearch/v7/esapi"
 	"github.com/gogo/protobuf/protoc-gen-gogo/generator"
+	"github.com/hashicorp/hcl/hcl/strconv"
```

Comment:
I think [`Unquote`](https://golang.org/pkg/strconv/#example_Unquote) is in the standard lib `strconv` package:

```suggestion
	"strconv"
```